includes/class-wcpdf-settings-upgrade.php

```diff
@@ -0,0 +1,95 @@
+<?php
+namespace WPO\WC\PDF_Invoices;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly
+}
+
+if ( !class_exists( '\\WPO\\WC\\PDF_Invoices\\Settings_Upgrade' ) ) :
+
+class Settings_Upgrade {
+
+	function __construct()	{
+		add_action( 'wpo_wcpdf_after_settings_page', array( $this, 'extension_overview' ), 10, 2 );
+	}
+
+	public function extension_overview( $tab, $section ) {
+		if ( $tab === 'upgrade' ) {
+			
+			$features = array(
```

Comment:
We could mention the extra settings for the packing slips, like the attachment option.

Reply:
Good idea.